src/Interfaces/IStargate.sol

```diff
@@ -1,8 +1,10 @@
-// Interface for Stargate V2
-/// @custom:version 1.0.0
-// SPDX-License-Identifier: BUSL-1.1
+// SPDX-License-Identifier: MIT
 pragma solidity ^0.8.17;
 
+/// @title Interface for StargateV2
+/// @author LI.FI (https://li.fi)
+/// @custom:version 1.0.0
```

Comment:
shouldn't it be right above `interface IStargate {` line?

Reply:
yeah I was a bit unsure because we have these two structs sitting above and its weird that the title comes after the structs...actually I believe the structs could be removed but that would trigger the audit checker. 
Thinking about it, I believe the interface files should be excluded from the audit checker...

For now I will move the title above the line you mentioned to be coherent